src/common/translations/localize.ts

```diff
@@ -12,7 +13,7 @@ export interface FormatsType {
   time: FormatType;
 }
 
-if (!Intl.PluralRules) {
+if (shouldPolyfill()) {
   import("@formatjs/intl-pluralrules/polyfill-locales");
```

Comment:
Should we await this?

Reply:
I would love top level await right now...

Reply:
Just enable it 😆 

https://webpack.js.org/blog/2020-10-10-webpack-5-release/#experiments

Reply:
I tried, but Babel doesnt support it yet...

Reply:
Oh wait... https://babeljs.io/docs/en/babel-plugin-syntax-top-level-await